eth_pydantic_types/address.py

```diff
@@ -14,7 +14,7 @@ def address_schema():
     return str_schema(min_length=42, max_length=42, pattern=ADDRESS_PATTERN)
 
 
-class Address(HashStr20):
```

Comment:
So `HashStr` is actually more like a `HexStr` because we do validation that it is a valid Hex.

- [ ] Rename `String` to `HexStr` (not the ABI one , the normal one)

Reply:
we should talk about either combining the string.py and hex.py (which has a HexStr) classes or pulling the hex value validation out of string.py so it can be a general use string type. really it's the HexStr that should be used by address and needs the explicit size in the class, not String